include/boost/geometry/util/range.hpp

```diff
@@ -116,10 +116,10 @@ front(Range const& rng)
 
 /*!
 \brief Short utility to conveniently return the front element of a Range.
-\ingroup utility
-*/
+ \ingroup utility
+ */
```

Comment:
I still see only one commit in this PR.

As far as I understand `git commit --amend` works for the tip of the tree. So, since you have a single commit you can certainly use it in your local branch to fix things in the commit message.
To actually redo the commit, you probably need to do `git reset <some hash>` or `git reset <some reflog>`, do the modifications and redo the commit.
All these are okay with your local branch, but I have no experience as to how they play along with pull requests.